types/react-native-material-textfield/index.d.ts

```diff
@@ -64,6 +63,15 @@ export interface TextFieldProps extends TextInputProps {
 
     onPress?(event: Event): void;
     onChangeText?(text: string): void;
+
+    renderLeftAccessory?(): JSX.Element;
+    renderRightAccessory?(): JSX.Element;
+
+    lineType?: string;
+    disabledLineType?: string;
```

Comment:
Should be https://github.com/n4kz/react-native-material-textfield/blob/master/src/components/line/index.js#L8

Reply:
Updated